### PR TITLE
Update to latest Arcade 7.0

### PR DESCRIPTION
**Pull request by MattGal**
(outcome of "darc update-dependencies --id=153321") to pick up -Svc pool behaviors and new build agent names

### CODE REVIEW

**Review by Tratcher — DISMISSED** [review]
Does this need to happen in all of our release branches?

**Comment by Tratcher** [comment]
> .dotnet\sdk\7.0.100-rc.2.22477.23\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): error NETSDK1138: (NETCORE_ENGINEERING_TELEMETRY=Restore) The target framework 'net5.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy.

The release branches still track our 1.x releases that supported 5.0. This error will need to be suppressed.

**Comment by MattGal** [comment]
> Does this need to happen in all of our release branches?

Ideally yes

**Comment by MattGal** [comment]
@Tratcher looks like to make this work it also needs YAML steps to preinstall a 6.0 runtime .  Is this branch still actually in use?

**Comment by Tratcher** [comment]
Yes, our release branches are used for servicing.

File: src/TelemetryConsumption/Yarp.Telemetry.Consumption.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Yarp.ReverseProxy extension package for in-process telemetry consumption</Description>
-    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
```
Comment (Tratcher) [inline src/TelemetryConsumption/Yarp.Telemetry.Consumption.csproj:5]: Removing TFMs is a breaking change we can't make in a servicing branch.

Comment (MattGal) [reply src/TelemetryConsumption/Yarp.Telemetry.Consumption.csproj]: I set CheckEolTargetFramework instead globally to allow this, should be clean now

**Comment by MattGal** [comment]
@Tratcher please re-review when you get a chance.

**Review by Tratcher — APPROVED** [review]

**Pull request merged**